src/inkbird_ble/parser.py

```diff
@@ -16,6 +16,8 @@
 from home_assistant_bluetooth import BluetoothServiceInfo
 from sensor_state_data import SensorLibrary
 
+IAMT1_SERVICE_UUID = "0000ffe4-0000-1000-8000-00805f9b34fb"
```

Comment:
I don't believe this is the right service. The device I have uses service FFE0 with **characteristic** FFE4.